src/DatabaseManager.php

```diff
@@ -215,12 +152,7 @@ public function rollBack()
         return true;
     }
 
-    /**
-     * @param string|null $name
-     *
-     * @return mixed
-     */
-    public function getLastInsertId($name = null)
+    public function getLastInsertId(string $name = null)
```

Comment:
I'm a bit surprised. It really shouldn't be allowed at all at the language level. But if they allow it, ok.